backend/app/modules/imagefetcher/FetcherActor.scala

```diff
@@ -85,30 +88,36 @@ class FetcherActor(ws: WSClient, serverUrl: String, monitoring: ActorRef) extend
   }
 
   /**
-    * Download a file from an url through an akka stream an place it a file
-    * (Downloaded file is not labeled yet)
+    * Download a file from an url through an akka stream an place it in the downloaded folder
+    * (Downloaded file is not labeled yet e.g random name)
```

Comment:
By the way, you may want to use the route parameter of the image to store the picture. Two generation of the same picture will generate the same route as much as I tested.